src/backend/joanie/core/serializers/admin.py

```diff
@@ -161,13 +161,86 @@ def validate(self, attrs):
         return validated_data
 
 
+class AdminOrganizationAddressSerializer(serializers.ModelSerializer):
+    """Serializer for the Address model for an organization"""
+
+    class Meta:
+        model = models.Address
+        fields = [
+            "id",
+            "title",
+            "address",
+            "postcode",
+            "city",
+            "country",
+            "first_name",
+            "last_name",
+            "is_main",
+            "is_reusable",
+        ]
+        read_only_fields = [
+            "id",
+        ]
+
+    def create(self, validated_data):
+        """
+        Create a new address and attach it to the provided organization.
+        """
+        validation_error = {}
+        organization_id = self.context.get("organization_id", None)
+
+        if organization_id is None:
+            validation_error["organization_id"] = "This field is required"
+
+        if validation_error:
+            raise serializers.ValidationError(validation_error)
+
+        try:
+            organization = models.Organization.objects.get(pk=organization_id)
+        except models.Organization.DoesNotExist as exception:
+            raise serializers.ValidationError(
+                {"organization_id": "Resource does not exist."}
+            ) from exception
+
+        validated_data["organization"] = organization
```

Comment:
I wonder if we have to do that 🤔 
Did you try something like that ? 

```suggestion
validated_data["organization_id"] = organization_id
```